lib/internal/errors.js

```diff
@@ -204,6 +206,15 @@ const addCodeToName = hideStackFrames(function addCodeToName(err, name, code) {
   }
 });
 
+function isErrorStackTraceLimitWritable() {
+  const desc = ObjectGetOwnPropertyDescriptor(Error, 'stackTraceLimit');
+  if (desc === undefined) {
+    return ObjectIsExtensible(Error);
+  }
+
+  return desc.writable ?? typeof desc.set === 'function';
```

Comment:
These need to be `own` checked since descriptors inherit from Object

Reply:
Something like that?
```suggestion
  return ObjectHasWonProperty('writable') ? desc.writable : ObjectHasWonProperty('set');
```

Reply:
yea